src/client/templates/cardModal.html

```diff
@@ -0,0 +1,61 @@
+<template name="cardModal">
+  {{#with cardModalInfo this.id}}
+  <div class="modal" id="{{_id}}" tabindex="-1" role="dialog">
+    <div class="modal-dialog" role="document">
+      <div class="modal-content">
+        <div class="modal-header">
+          <button type="button" class="close" data-dismiss="modal"><span aria-hidden="true"><i class="fa fa-times"></i></span></button>
+          <h4 class="modal-title">{{author.profile.name}}</h4>
+        </div>
+        <div class="modal-body">
+          <p>Thought: {{text}}</p>
+          <p>Category: {{category}}</p>
+          <p>Tags: {{#each tags}} {{this}}, {{/each}} </p>
```

Comment:
This will always add an extra comma at the end of the tags (if there is at least 1)